src/systems/sensors/Sensors.cc

```diff
@@ -555,6 +561,11 @@ std::string Sensors::CreateSensor(const Entity &_entity,
     sensor = this->dataPtr->sensorManager.CreateSensor<
       sensors::ThermalCameraSensor>(_sdf);
   }
+  else if (_sdf.Type() == sdf::SensorType::BOUNDINGBOX_CAMERA)
+  {
+    sensor = this->dataPtr->sensorManager.CreateSensor<
+      sensors::BoundingBoxCameraSensor>(_sdf);
```

Comment:
I have a problem with integrating the Segmentation & Boundingbox sensors from `ign-sensors` in `ign-gazebo` .. it was working before I merge the branch with the `main branch` .. now when i run any SDF file with the segmentaiton or boundingbox sensors it gives me error like this 
```
ign gazebo server: symbol lookup error: /home/amrelsersy/ignition/install/lib/ign-gazebo-6/plugins/libignition-gazebo-sensors-system.so: undefined symbol: _ZN8ignition7sensors2v624BoundingBoxCameraSensorC1Ev
```
in that line .. and I already include the sensor in that file

....
note that the ign-sensors::BoundingBoxCameraSensor itself is working(its integration test passes) and also the ign-rendering::BoundingBoxCamera is working
....

all the repos (sensors, rendering, gazebo ... etc) is updated with the main branch of all of them